packages/features/insights/filters/EventTypeList.tsx

```diff
@@ -14,7 +16,7 @@ const mapEventTypeToOption = (eventType: EventType): Option => ({
   label: eventType.teamId ? `${eventType.title} (${eventType.team?.name})` : eventType.title,
 });
 
-export const EventTypeList = () => {
+export const EventTypeList = memo(() => {
```

Comment:
Parents were causing this to re-render on prop change - we didnt need this as all state is internal here